mackup/application.py

```diff
@@ -136,9 +242,9 @@ def backup(self) -> None:
                 else:
                     print("Doing nothing\n  {}\n  does not exist".format(home_filepath))
 
-    def restore(self) -> None:
+    def link(self) -> None:
```

Comment:
And maybe add a runtime warning or a confirmation dialog too on macOS Sonoma and later

Reply:
Yes, it's super annoying, and my guess is that it's related to what the terminal running mackup can do... 🤷‍♂️